api/src/main/java/org/openmrs/module/sync2/api/sync/SyncClient.java

```diff
@@ -113,30 +113,42 @@ private void prepareRestTemplate(ClientHelper clientHelper) {
 
 	private Object retrieveObject(String category, String url, ClientHelper clientHelper)
 			throws RestClientException, URISyntaxException {
+		/** TODO: Restore wrapping, when the code for receiving wrapped request will be finished
```

Comment:
I guess those changes were only for testing purposes

Reply:
Yes, I already solved it.